Makefile

```diff
@@ -106,34 +114,44 @@ css: $(BUILD_CSS)
 font: $(FONT_TARGET)
 clean:
 	rm -rf $(BUILD_DIR)
-
-$(PJS_SRC): $(NODE_MODULES_INSTALLED)
+# This adds an entry to your local .git/config file that looks like this:
+# [include]
+# 	path = ../.gitconfig
+# that tells git to include the additional configuration specified inside the .gitconfig file that's checked in here.
+setup-gitconfig:
+	@git config --local include.path ../.gitconfig
```

Comment:
Note, running `make setup-gitconfig` once will make blame skip the commits where prettier was run, which helps a lot with preserving history.

We could consider documenting this somewhere more prominent, or trying to run this as part of install, or something like that.